src/ResultInterface.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ * Zend Framework (http://framework.zend.com/)
+ *
+ * @link      http://github.com/zendframework/zf2 for the canonical source repository
+ * @copyright Copyright (c) 2005-2015 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license   http://framework.zend.com/license/new-bsd New BSD License
+ */
+
+namespace Zend\Validator;
+
+/**
+ * This interface provide methods for to know the result of a validation and the reasons when the result is not valid
+ *
+ * This interface resolve the following questions:
+ *    - Is the result valid or invalid?
+ *    - If not, Why is not valid?
+ */
+interface ResultInterface
+{
+    /**
+     * Is the validation result valid?
+     *
+     * @return bool
+     */
+    public function isValid();
+
+    /**
+     * Is the validation result invalid?
+     *
+     * @return bool
+     */
+    public function isNotValid();
```

Comment:
I see it close like when you have `getFields` and `getField($name)` with the first method you can do the same operations as the second, but the second is more user friendly.

Also the implementation is trivial, the return is just the alternate version of isValid (!isValid)

For me expressiveness is human level of write. (!) not operator is not in the dictionary :)